CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## [Unreleased]
```

Comment:
this needs also a section at the bottom of the page with a valid github compare link

Reply:
`[Unreleased]: https://github.com/owncloud/files_primary_s3/compare/v1.2.0...master`

Reply:
can you add that line?